src/integration/common/SEO.js

```diff
@@ -192,5 +192,25 @@ export default () => {
         });
       });
     });
+
+    describe('Apple Touch Icon', () => {
+      const touchIconScript = document.querySelectorAll(
+        'head > script[rel="apple-touch-icon"]',
+      );
+
+      touchIconScript.forEach(script => {
+        const url = script.getAttribute('href');
+        const sizes = script.getAttribute('sizes');
+
+        it('should be in the document', () => {
+          expect(script).toBeInTheDocument();
+        });
+
+        it(`should match attributes`, () => {
```

Comment:
Could we try something like this (~stolen~ borrowed from another PR!) - https://github.com/bbc/simorgh/blob/2bd4daaa2962fbf91a59c1f02a277272ff3bdfed/src/integration/common/footer.js#L42

```suggestion
        it(`should match attributes`, () => {
          expect({
          url,
          sizes
          }).toMatchSnapshot();
        };
```